flathub.json

```
@@ -0,0 +1,3 @@
+{
+    "skip-arches": ["aarch64", "arm"]
```

```suggestion
    "only-arches": ["x86_64"]
```

But if you build from source, this can be avoided.

---

This *file* should be removed, otherwise aarch64 won't be built